Directory.Build.props

```diff
@@ -1,9 +1,9 @@
 <Project>
 
   <PropertyGroup>
-    <Version>0.89.4</Version>
-    <AssemblyVersion>0.89.4</AssemblyVersion>
-    <FileVersion>0.89.4</FileVersion>    
+    <Version>0.90.0</Version>
```

Comment:
I need you to back this change out. We don't want to bump to 0.90 yet.

Reply:
No problem. Do you'll submit a PR to this change and I'll rebase or do you want I made the change?

Reply:
Just modify this PR to use 0.89.4.

Reply:
I'm finishing some more fixes and improvements and I'll committing when everything is ready. Thanks.